docs/EN_US/ContainerizedHPCC/ContainerizedMods/ConfigureValues.xml

```diff
@@ -857,6 +857,81 @@
       </sect3>
     </sect2>
 
+    <sect2 id="Egress-HYAML" role="brk">
+      <title>Egress</title>
+
+      <para>In order to allow clusters to be securely locked down but still
+      allow access to the services they need there is an egress mechanism.
+      Egress provides a similar mechanism to Ingress, by being able to define
+      which endpoints and ports components are permitted to connect to.</para>
+
+      <para>Most HPCC Systems components have their own auto-generated network
+      policies. The generated network policies typically work to limit ingress
+      and egress to inter-component communication, or expected external
+      service ingress only.</para>
+
+      <para>For instance, in a default deployed system with network policies
+      enforced, a query running (on hThor, Thor, or Roxie), will not be able
+      to connect with a 3rd party service, such as an LDAP service or log
+      stack. </para>
+
+      <para>Deploying generates a "default-deny-all-but-dns" which prevents
+      all but DNS, however, we also generate "allow-apiserver" (also with no
+      pod-selector).</para>
```

Comment:
I'd remove this sentence. It doesn't flow from the previous text, and it doesn't really help to know the internal details we generate like this in abstraction.